pkgs/os-specific/linux/hdparm/default.nix

```diff
@@ -1,22 +1,22 @@
 { stdenv, fetchurl }:
 
 stdenv.mkDerivation rec {
-  name = "hdparm-9.48";
+  name = "hdparm-9.51";
 
   src = fetchurl {
     url = "mirror://sourceforge/hdparm/${name}.tar.gz";
-    sha256 = "1vpvlkrksfwx8lxq1p1nk3ddyzgrwy3rgxpn9kslchdh3jkv95yf";
+    sha256 = "14ax5lyzhigx58ing7adbfyzisv0fqajbmzphg149rnb3s4xiyhs";
   };
 
-  preBuild = ''
-    makeFlagsArray=(sbindir=$out/sbin manprefix=$out)
-  '';
+  installPhase = ''
+    make DESTDIR=$out install
```

Comment:
You omitted `manprefix` here. This causes the manpages to be installed to `$out/usr/share/man`.